docs/api/config/js_kanban_columnshape_config.md

```diff
@@ -143,7 +151,35 @@ const columnShape = {
         ]
     },
     fixedHeaders: false,
-    css: (column, cards) => column.id == "inprogress" && cards.length < 5 ? "green" : "red"
+    css: (column, cards) => column.id == "inprogress" && cards.length < 5 ? "green" : "red",
+    headerTemplate: template(column => {
```

Comment:
the example is impressive. mb too impressive. we should simplify it for the docs, I guess 

```js
`<div class="wx-collapse-icon" data-action=${"collapse"}>
         <i class=${column.column.collapsed ? "wxi-angle-right" : "wxi-angle-left"}></i>
 </div>
                ${
                    !column.column.collapsed
                        ?   `<div class="wx-label" data-action="rename">
                                ${escapeHTML(column.column.label)}
                                (${column.columnState.cardsCount})
                            </div>`
                        : ""
                }
                ${
                    !column.column.collapsed
                        ?   `<div class="wx-menu" data-menu-id={column.id}>
                                <i class="wxi-dots-h"></i>
                            </div>`
                        : ""
                }`;
```